src/TestFramework/PhpUnit/CommandLine/ArgumentsAndOptionsBuilder.php

```diff
@@ -80,10 +96,11 @@ public function buildForInitialTestsRun(string $configPath, string $extraOptions
 
     /**
      * @param TestLocation[] $tests
+     * @return list<string>
      */
     public function buildForMutant(string $configPath, string $extraOptions, array $tests, string $testFrameworkVersion): array
     {
-        $options = $this->buildForInitialTestsRun($configPath, $extraOptions);
+        $options = $this->prepareArguments($configPath, $extraOptions);
```

Comment:
what is that change about exactly?

Reply:
previously, `buildForMutant` was using public `buildForInitialTestsRun()` method and on top of it added more arguments/options.

Now, after `buildForInitialTestsRun()` has it's own logic for `--filter`, that should be *only* for "Initital Tests Run" stage, we can't reuse it as before.

So I extracted the common reusable part to `prepareArguments()` private helper method, and reused it in both `buildForInitialTestsRun()` and `buildForMutant()`.